### ISSUE TITLE
Devices [IG]

### ISSUE THREAD

**Issue by BSnoeijerCD** [cdisc-org/DDF-RA#518]
See feature ticket #442

Devices are a separate part of IDMP and will be handled after the interventions part.
See #316 for including Interventions in the USDM model according to IDMP.

We can include #508 (Product designation) and #259 (Other intervention names) with changes based on this ticket.

**Comment by BSnoeijerCD** [comment]
As discussed during scrum today: difference between products not separatable from device and other products (like refills, caps etc) should be made clear in the IG regarding how this should be handled (differently) in the model.

**Comment by BSnoeijerCD** [comment]
@dih-cdisc 
I updated the UML and text in the IG. Are you ok to add the below instance diagram with corresponding explanation?
And if so: any suggestions for improvements? I did not add all the attribute instances. Doing all might make it messy but to little can also be confusing.
<img width="974" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOVhSQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c14a6dac147a93976e2dc4548f685dedfe38e789/Device_2_instance_2.png" alt="Device_2_instance_2.png" />
